packages/tools-iottwinmaker/src/lib/policy/workspace-role-assume-policy.ts

```diff
@@ -1,12 +1,18 @@
-export const WORKSPACE_ROLE_ASSUME_POLICY = {
-  Version: '2012-10-17',
-  Statement: [
-    {
-      Effect: 'Allow',
-      Principal: {
-        Service: ['iottwinmaker.amazonaws.com'],
+export const getWorkspaceRoleAssumePolicy = (isGamma: boolean) => {
+  return {
+    Version: '2012-10-17',
+    Statement: [
+      {
+        Effect: 'Allow',
+        Principal: {
+          Service: [
+            isGamma
+              ? 'iottwinmaker.aws.internal'
```

Comment:
similar comment: probably shouldn't publish this in the code

can we instead have a way for to provide a custom workspace role policy if needed?